front/src/assets/rollingStock/freightRollingStocks.ts

```diff
@@ -108,6 +108,11 @@ const FREIGHT_ROLLING_STOCKS = [
   'Y8000AG',
   'Y8000AP',
   'Y9000US',
+  'electric_rolling_stock_test_e2e',
+  'dual-mode_rolling_stock_test_e2e',
+  'slow_rolling_stock_test_e2e',
+  'fast_rolling_stock_test_e2e',
+  'improbable_rolling_stock_test_e2e',
```

Comment:
those rollingstocks are only used for tests, can't we declare them in an asset file in `tests/` then spread them in this array only if the tests are started?